docker/conf/docker-compose-mac.yml.dist

```diff
@@ -24,12 +24,11 @@ services:
             - smtp-server
 
     php-fpm:
-        build: project-base/docker/php-fpm
+        image: shopsys/framework-php-72-fpm:alpha-1
```

Comment:
Why call it `shopsys/framework-php-72-fpm` and not just `shopsys/php-fpm`? Also, I think that the version should not contain just the *alpha1* part, but full version, ie. *7.0.0-alpha1*.